src/client/oicp/CpoOICPClient.ts

```diff
@@ -0,0 +1,1303 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
Never ever disable linter rules at a file level. Fix your code or do it at the line level.

Reply:
okay. I deleted it